### DISCUSSION TITLE
0.6.4 - Brand new overlay

### DISCUSSION THREAD

**Discussion by XorTroll** [XorTroll/emuiibo#201] [General]
# Changelog

## emuiibo

- Fixed some minor issues related to virtual amiibo area saving

- Fixed a weird internal issue where sometimes emuiibo would become unaccessible under certain circumstances

## overlay

The overlay has almost entirely changed thanks to the huge work of **AD2076** and **AmonRaNet**:

- Really improved UI and design

- Now, aside from English, the following languages are supported: German, Spanish, Russian, Traditional and Simplified Chinese

## emutool

No changes (not uploading a new release)

<hr /><em>This discussion was created from the release <a href='https://github.com/XorTroll/emuiibo/releases/tag/'>0.6.4 - Brand new overlay</a>.</em>

**Comment by ta-sho** [comment]
Hello, I see that few people argue in this way,... Ask how to insert the image of the amiibo in the tesla?,... without further ado, thanks for the work you do

**Comment by Lexi-Pixel** [comment]
I apologize if this isn't the place to ask, but I've been at this for a few hours now.

I got all of this working beautifully, save for one detail. If emuiibo no longer accepts raw .bin dumps anymore, how do I get wolf link to be maxed out in BotW? I tried learning a way to manually edit the one emutool makes, but I can't work it out. I'd appreciate any response. I have a maxed out .bin already, but don't know how to use it ingame virtually.

Any help is mega appreciated

**Comment by impeeza** [comment]
Easy:  reading the Readme.MD on the main page of the repo:

https://github.com/XorTroll/emuiibo#amiibo-format

there is a section dedicated to explicit that matter,   please remember this is a space for bug report not seek for help.

**Reply by Lexi-Pixel** [reply]
Thank you for the help, I'm embarrassed it's on the readme, but with my advanced dyslexia, the text can be really tough to parse :) 

I super appreciate you pointing me too it even though it was obvious! Thank you! I got it all working
